exporter/datadogexporter/go.mod

```diff
@@ -52,6 +52,7 @@ require (
 	github.com/Showmax/go-fqdn v1.0.0 // indirect
 	github.com/antonmedv/expr v1.9.0 // indirect
 	github.com/armon/go-metrics v0.3.10 // indirect
+	github.com/benbjohnson/clock v1.3.0 // indirect
```

Comment:
Why is this being added ?

Reply:
This got added when I imported `"go.uber.org/zap/zaptest"` in `translator_test.go`